pkg/services/sqlstore/apikey.go

```diff
@@ -42,12 +42,15 @@ func (ss *SQLStore) GetAPIKeys(ctx context.Context, query *models.GetApiKeysQuer
 	})
 }
 
-// GetAllOrgsAPIKeys queries the database for valid non SA APIKeys across all orgs
-func (ss *SQLStore) GetAllOrgsAPIKeys(ctx context.Context) []*models.ApiKey {
+// GetAllAPIKeys queries the database for valid non SA APIKeys across all orgs
+func (ss *SQLStore) GetAllAPIKeys(ctx context.Context, orgID int64) []*models.ApiKey {
```

Comment:
should we enforce this function to be only `orgID` specific. So that if we would have a `orgID < 1` we return an error, to specify a `orgID`